### ISSUE TITLE
[Image needs adding] Data Stewards Chapter

### ISSUE THREAD

**Issue by aleesteele** [the-turing-way/the-turing-way#3375]
### Summary

The chapter: https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles/data-steward.html does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

![datastewards](https://github.com/the-turing-way/the-turing-way/assets/18509789/f92bb847-f44d-4833-95c8-0a56fc03af0b)

### What needs to be done?

- [ ] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [ ] Write the alt text for the image
- [ ] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

**Comment by kallewesterling** [comment]
Addressed in #3379.
